system-tests/docker-compose.yml

```diff
@@ -86,11 +101,40 @@ services:
     environment:
       API_KEY: ApiKeyDefaultValue
       EDC_HOST: provider
-      ASSETS_STORAGE_ACCOUNT: ${PROVIDER_ASSETS_STORAGE_ACCOUNT}
+      ASSETS_STORAGE_ACCOUNT: providerassets
     depends_on:
       consumer-eu:
         condition: service_healthy
       consumer-us:
         condition: service_healthy
       provider:
-        condition: service_healthy
\ No newline at end of file
+        condition: service_healthy
+
+  did-server:
```

Comment:
it might help to comment what each services does, especially for new adopters

Reply:
Good suggestion, updated. Thanks.